src/Controllers/API/EditApplicantController.php

```diff
@@ -61,13 +61,21 @@ public function __invoke(Request $request, Response $response, array $args)
                     $newApplicationData['laptopDeposit'],
                     $newApplicationData['laptopNum'],
                     $newApplicationData['taster'],
-                    $newApplicationData['tasterAttendance']
+                    $newApplicationData['tasterAttendance'],
+                    null,
+                    $newApplicationData['stageId'],
+                    $newApplicationData['optionId']
                 );
+                //If applicant is being moved to a stage that makes them a student a row in the
+                // applicant_additional table needs to be created. If it doesn't already exist.
+
```

Comment:
Is there a need for two lines breaks here? If not, please remove :)

Reply:
Also on lines 69 and 70, if the comments do not relate to the code above, add a line break there too (between lines 69 and 70).